spec/factories/event_schedule.rb

```diff
@@ -2,6 +2,7 @@
 
 FactoryGirl.define do
   factory :event_schedule do
+    start_time DateTime.parse("#{Date.today.to_s} 00:00").utc
```

Comment:
The problem with parse is that after you change it to utc, so the time would also change, and won't be 10:00. And it could happens that it is outside of the conference time too